core-bundle/src/EventListener/DataContainer/LogoutPageRedirectListener.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\DataContainer;
+
+use Contao\CoreBundle\DependencyInjection\Attribute\AsCallback;
+use Contao\DataContainer;
+use Contao\Module;
+
+#[AsCallback(table: 'tl_page', target: 'fields.jumpTo.attributes')]
+class LogoutPageRedirectListener
+{
+    /**
+     * @param DataContainer|Module|null $dc
```

Comment:
In which case is `$dc` a module instance?

Reply:
That's from `Widget::getAttributesFromDca()`. It can actually be anything, e.g. for Haste Form it will be something else, I think some of our front end modules (e.g. personal data) pass the module here.